docs/kyma-environment-broker/08-02-deprovisioning-kyma-environment.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Deprovision Kyma environment through Kyma Environment Broker
+type: Tutorials
+---
+
+This tutorial shows how to deprovision Kyma environment on Azure.
+
+## Steps
+
+1. Ensure these environment variables are exported:
+```bash
+export BROKER_URL={KYMA_ENVIRONMENT_BROKER_URL}
+export INSTANCE_ID={INSTANCE_ID_FROM_PROVISIONING_CALL}
+```
+
+2. Obtain Authorization credentials as described in [Authorization section](./03-05-authorization.md). Export this variable based on chosen authorization method:
+
+```bash
+export AUTHORIZATION_HEADER="Authorization: Basic $ENCODED_CREDENTIALS"
```

Comment:
Shouldn't it be `Authorization: {Basic OR Bearer}` as in the provisioning tutorial?